packages/contracts/contracts/foundry/SphinxUtils.sol

```diff
@@ -50,6 +51,44 @@ contract SphinxUtils is SphinxConstants, StdUtils {
         return _data[_start:_end];
     }
 
+    function checkAccesses(Vm.AccountAccess[] memory accountAccesses, bytes32 creationCodeHash, bytes32 runtimeCodeHash) public view returns (bool) {
+        // If there aren't two calls, (one to the deployment proxy, and another to deploy the contract)
```

Comment:
We have to pass in the creation and runtime code because `Sphinx.sol` (which deploys the test contract) is compiler by the users machine whereas `SphinxUtils.sol` is compiled by us and injected with raw bytecode. So if we tried to get the creation and runtime code directly from `SphinxForkTest` within `SphinxUtils.sol` it would not match the code for the corresponding contract in `Sphinx.sol`.